src/main/verifyIso.ts

```diff
@@ -9,91 +9,198 @@ interface IsoHashInfo {
 
 const isoHashes = new Map<string, IsoHashInfo>();
 
-// Valid ISOs
+// Valid Brawl ISOs
+
+isoHashes.set("0e95949ac585f357e79fcd34b20670b5dca97ac2", {
+  valid: IsoValidity.VALID,
+  name: "unkown Brawl copy sha1 hash",
+});
+isoHashes.set("d18726e6dfdc8bdbdad540b561051087", {
+  valid: IsoValidity.VALID,
+  name: "unknown Brawl copy MD5 hash",
+});
+
+isoHashes.set("d8560b021835c9234c28be7ff9bcaaeb", {
+  valid: IsoValidity.VALID,
+  name: "Brawl alt 1 MD5 hash",
+});
+
+isoHashes.set("5052e2e15f22772ab6ce4fd078221e96", {
+  valid: IsoValidity.VALID,
+  name: "Brawl alt 2 MD5 hash",
+});
+
+isoHashes.set("52ce7160ced2505ad5e397477d0ea4fe", {
+  valid: IsoValidity.VALID,
+  name: "Brawl alt 3 MD5 hash",
+});
+
+isoHashes.set("9f677c78eacb7e9b8617ab358082be32", {
+  valid: IsoValidity.VALID,
+  name: "Brawl alt 4 MD5 hash",
+});
+
+isoHashes.set("1c4d6175e3cbb2614bd805d32aea7311", {
+  valid: IsoValidity.VALID,
+  name: "Brawl alt 5 MD5 hash",
+});
+
+// Known Invalid ISOs
+
+isoHashes.set("0635cfd4730205c1688bf3ba34237eef", {
```

Comment:
Instead of random ISOs, you may want to add the MD5s of baked PM 3.6/other common modded ISOs (Minus and Smash 2 comes to mind). I don't have them on hand, but they exist and will likely cause a crash.